### PR TITLE
BAH-849: Enable DEBUG option in odoo-connect

### PR DESCRIPTION
**Pull request by chinzou**
https://bahmni.atlassian.net/browse/BAH-849

### CODE REVIEW

File: odoo-connect/startup.sh [hunk 1]
```diff
@@ -11,5 +11,10 @@ DEBUG=${DEBUG:-false}
 sleep 3
 /etc/bahmni-erp-connect/run-liquibase.sh
 
+if [ $DEBUG ]; then
+    export JPDA_ADDRESS="1044"
+    export JPDA_TRANSPORT=dt_socket
+fi
+
 # start tomcat
 /usr/local/tomcat/bin/catalina.sh jpda run
```
Comment (rbuisson) [inline odoo-connect/startup.sh]: `jpda`enables the actual debug mode. Could this be conditional too?

File: odoo-connect/startup.sh [hunk 2]
```diff
@@ -11,10 +11,13 @@ DEBUG=${DEBUG:-false}
 sleep 3
 /etc/bahmni-erp-connect/run-liquibase.sh
 
-if [ $DEBUG ]; then
+if [ $DEBUG == true ]; then
     export JPDA_ADDRESS="1044"
```
Comment (rbuisson) [inline odoo-connect/startup.sh]: Maybe that'd be ok to keep the default Tomcat port here. Which is `8000`.

File: docker-compose.yml [hunk 3]
```diff
@@ -142,7 +142,7 @@ services:
       OPENMRS_HOSTNAME: "openmrs-tomcat"
       OPENMRS_USERNAME: "admin"
       OPENMRS_PASSWORD: "${OPENMRS_PASSWORD:-Admin123}"
-      BAHMNI_LOGIN_URL: "http://localhost/"
+      BAHMNI_LOGIN_URL: "http://localhost/bahmni/home/index.html#/login"
```
Comment (rbuisson) [inline docker-compose.yml]: this line is not necessary. this creates conflicts with the upstream. Could you fix it?

Comment (chinzou) [reply docker-compose.yml]: ah yes :)

**Pull request merged**